snow/networking/sender/sender.go

```diff
@@ -894,7 +894,13 @@ func (s *sender) SendPut(_ context.Context, nodeID ids.NodeID, requestID uint32,
 // The PushQuery message signifies that this consensus engine would like each
 // node to send their preferred frontier given the existence of the specified
 // container.
```

Comment:
This seems outdated with this PR (also accepted ID is not documented). Or just remove the old comment? The code itself is obvious to me.

Reply:
> // SendChits sends chits

Yeah gunna remove these lol